libr/arch/p/m680x_cs/plugin.c

```diff
@@ -525,13 +544,38 @@ static bool decode(RArchSession *as, RAnalOp *op, RAnalOpMask mask) {
 static char *regs(RArchSession *as) {
 	const char *p = \
 		"=PC    pc\n"
-		"=SP    sp\n"
-		"=A0    a0\n"
-		"=A1    a1\n"
-		"gpr	pc	.16	48	0\n"
-		"gpr	sp	.16	48	0\n"
-		"gpr	a0	.16	48	0\n"
-		"gpr	a1	.16	48	0\n";
+		"=SP    s\n"
+		"gpr	pc	.16	0	0\n" ///< M6800/1/2/3/9, M6301/9
+		"gpr	s	.16	2	0\n" ///< M6809/M6309 system stack (=sp on others)
+		"gpr	cc	.8	4	0\n" ///< M6800/1/2/3/9, M6301/9
+		"flg	C		.1	4.0	0\n"
+		"flg	V		.1	4.1	0\n" // At least 6805 lacks it? Are the others shifted??
```

Comment:
two tabs. should be one

Reply:
I probably copied that from some other place… fixed.